omod/src/main/resources/config.xml

```diff
@@ -27,27 +27,31 @@
 			org.openmrs.module.uilibrary
 		</require_module>
 
-		<require_module version="1.0.2">
+		<require_module version="1.0.0">
 			org.openmrs.module.mchapp
 		</require_module>
         
         <require_module version="2.4">
             org.openmrs.module.appframework
         </require_module>
 		
-		<require_module version="1.5.1">
+		<require_module version="1.0.0">
 			org.openmrs.module.hospitalcore
 		</require_module>
 
+		<require_module version="17.0.4">
```

Comment:
KenyaEMR version we have on the test server 17.0.3

Reply:
Actually sir its 17.0.4